Source/Core/Core/PowerPC/PowerPC.h

```diff
@@ -30,6 +30,29 @@ enum
   CORE_CACHEDINTERPRETER,
 };
 
+#ifdef _M_X86_64
+constexpr std::array<CPUCore, 4> AVAILABLE_CPU_CORES = {
```

Comment:
This array should likely not be placed verbatim wholesale in the header (i.e. both the definition and declaration shouldn't be in the header). This will cause it to be duplicated across all translation units its used within. It's also very messy from an API perspective (placing platform specifics in a header).

A better way of doing this would be to just use a `std::vector` in this case (it's effectively negligible given the amount of entries). That, or the definition of the table should be in the cpp file.

e.g. one way of keeping platform-specific stuff wholly internal to the translation unit would be:

```cpp
// h
const std::vector<CPUCore>& AvailableCPUCores();
CPUCore DefaultCPUCore();

// cpp
const std::vector<CPUCore>& AvailableCPUCores()
{
  static const std::vector<CPUCore> cpu_cores = {
      // Stuff here
  };

  return cpu_cores;
}

CPUCore DefaultCPUCore()
{
  // Stuff here
}
```